soroban-spec/Cargo.toml

```diff
@@ -14,17 +14,17 @@ rust-version = "1.69"
 stellar-xdr = { workspace = true, features = ["next", "std", "serde"] }
 base64 = "0.13.0"
 thiserror = "1.0.32"
-syn = {version="1.0",features=["full"]}
+syn = {version="2.0",features=["full"]}
 quote = "1.0"
 proc-macro2 = "1.0"
 itertools = "0.10.3"
-darling = "0.14.1"
-sha2 = "0.10.2"
+darling = "0.20.0"
+sha2 = "0.9.9"
```

Comment:
Why the downgrade of sha2?

Reply:
de-duplication -- dalek is on an older one. the differences are harmless. we'll move everything ahead once dalek finally publishes 2.0 (_very_ hopefully this summer before we go final, but hard to predict, they just came out of neglected-maintenance)